pkg/k8shandler/forwarding.go

```diff
@@ -134,10 +134,10 @@ func (clusterRequest *ClusterLoggingRequest) NormalizeForwarder() (*logging.Clus
 	} else {
 		if len(unready)+len(degraded) > 0 {
 			status.Conditions.SetCondition(condDegraded(logging.ReasonInvalid, "degraded pipelines: invalid %v, degraded %v", unready, degraded))
-			logger.Infof("ClusterLogForwarder degraded")
+			logger.Debug("ClusterLogForwarder degraded")
```

Comment:
since this ends up in the cluster conditions does having this in the logs buy you guys anything?

Reply:
This leads to my general question from slack regarding status and logging any of these messages.  What value does this provide if the result is reflected in a CR status?

Reply:
removing